scripts/apps/authoring/styles/authoring.scss

```diff
@@ -63,6 +63,12 @@
 .subnav--authoring {
   z-index: 99;
   background-color: $subnav-authoring;
+  .highlights-toggle, .marks-toggle {
+    .dropdown__menu {
+      max-height: 250px;
```

Comment:
LGTM, however, if you change max-height to `calc(100vh - 350px);` , that will possibly allow some more height for menu items